src/load/azext_load/data_plane/load_test/help.py

```diff
@@ -118,6 +118,28 @@
         az load test convert-to-jmx --load-test-resource sample-alt-resource --resource-group sample-rg --test-id sample-existing-test-id
 """
 
+helps[
+    "load test set-baseline"
+] = """
+type: command
+short-summary: Set a baseline test run for the test.
+examples:
+    - name: Set baseline test run.
```

Comment:
@saloniagrawal1997, please review the short-summary and name

Reply:
"Set a test run as the baseline for comparison with other runs in the test."